cip-drep-metadata/README.md

```diff
@@ -0,0 +1,231 @@
+---
+CIP: ?
+Title: DRep Metadata
+Category: ?
+Status: Proposed
+Authors:
+    - Thomas Upfield <thomas.upfield@iohk.io>
+Implementors: []
+Discussions:
+    - https://github.com/cardano-foundation/CIPs/pull/?
+Created: 2024-02-07
+License: CC-BY-4.0
+---
+
+<!-- Existing categories:
+
+- Meta     | For meta-CIPs which typically serves another category or group of categories.
+- Wallets  | For standardisation across wallets (hardware, full-node or light).
+- Tokens   | About tokens (fungible or non-fungible) and minting policies in general.
+- Metadata | For proposals around metadata (on-chain or off-chain).
+- Tools    | A broad category for ecosystem tools not falling into any other category.
+- Plutus   | Changes or additions to Plutus
+- Ledger   | For proposals regarding the Cardano ledger (including Reward Sharing Schemes)
+- Catalyst | For proposals affecting Project Catalyst / the Jörmungandr project
+
+-->
+## Acknowledgements
```

Comment:
We're standardising CIPs and CPSs which have more than one round of contributors and/or more than a modest size of text to link to the end... can you please follow the example of [CIP-1694](https://github.com/cardano-foundation/CIPs/blob/master/CIP-1694/README.md#abstract) to move this section to the end and (at your discretion, and following a general recommendation) to make a text comment at the end of the Abstract to link to it?

Reply:
I see the section is moved (in a later commit not linked back here); and without the link which is optional... so resolving this.

Reply:
done.